docs/UserGuide.md

```diff
@@ -227,6 +227,28 @@ Examples:
 
 [Scroll back to Table of Contents](#table-of-contents)
 
+### Select doctor
+- Command format: `sd INDEX`
+- What it does: This command filters the patients list and displays only the patients associated with the doctor at `INDEX`.
```

Comment:
Perhaps we could add that the doctor's information is shown on the right hand column too.